src/apps/rDSN.monitor/app_package/MonitorApp.py

```diff
@@ -449,11 +449,6 @@ def post(self):
             savedFile.write(raw_file)
             savedFile.close()
             
-            #TODO: shall we need some mechanism to detect if 7z are truely installed?
```

Comment:
both the app store and the deployment server need to unzip the package. The former is for easy navigation on the web site (I mean look into the package using raw directory list). The latter is for deployment. Since they may be deployed on different servers, so both are necessary.